app/models/concerns/user/low_balance_fraud_check.rb

```diff
@@ -29,7 +31,23 @@ def check_for_low_balance_and_probate(refunded_or_disputed_purchase_id)
     disable_refunds_and_put_on_probation! unless recently_probated_for_low_balance?
   end
 
+  def mark_compliant_if_balance_recovered!
+    return unless can_recover_from_automatic_fraud_probation?
+
+    content = "Marked compliant automatically on #{Time.current.to_fs(:formatted_date_full_month)} as balance is recovered to #{MoneyFormatter.format(LOW_BALANCE_RECOVERY_THRESHOLD, :usd, no_cents_if_whole: true, symbol: true)}"
+    mark_compliant!(author_name: LOW_BALANCE_FRAUD_CHECK_AUTHOR_NAME, content:)
+  end
+
   private
+    def can_recover_from_automatic_fraud_probation?
+      return false unless on_probation?
+
+      comments.with_type_on_probation
+              .order(created_at: :desc, id: :desc)
+              .first&.author_name == LOW_BALANCE_FRAUD_CHECK_AUTHOR_NAME &&
```

Comment:
Should we add this check? 

I scanned through the codebase and it seems like `LOW_BALANCE_FRAUD_CHECK_AUTHOR_NAME` is the only author which can probate an account. 

Seems redundant to me, lmk if I'm missing something